src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1063,6 +1070,11 @@ def __init__(self, parent=None, title="Colormap Dialog"):
         layoutScale.addWidget(self._autoScaleCombo)
         layoutScale.addStretch()
 
+        layoutSaturation = qt.QFormLayout()
```

Comment:
How does this layout stretch horizontally?
What about combining this with `layoutScale` to avoid extra margins?